### PR TITLE
Brought Resonate to Stable State

### PR DESCRIPTION
**Pull request by Aarush-Acharya**
## Description

Brought Resonate to stable state,

- Removed print statements 
- Migrated Appwrite Plugin to 12.0.4
- Added documentation for onboarding script

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] Refactor (does not change functionality, e.g. code style improvements, linting)

## How Has This Been Tested?

The following flows were tested 

- Login/Logout
- Email Verification
- Discussion Creation/Deletion/Conversion
- Room Creation/ Deletion
- Room mic on/off

Please include screenshots below if applicable.

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [X] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by chandansgowda** [comment]
Hey @vrundraval24 
please review.

File: lib/controllers/discussions_controller.dart [hunk 1]
```diff
@@ -81,10 +81,15 @@ class DiscussionsController extends GetxController {
   }
 
   Future<void> removeUserFromSubscriberList(String subscriberId) async {
-    await databases.deleteDocument(
-        databaseId: "6522fcf27a1bbc4238df",
-        collectionId: "6522fd267db6fdad3392",
-        documentId: subscriberId);
+    try {
+      await databases.deleteDocument(
+          databaseId: "6522fcf27a1bbc4238df",
```
Comment (vrundraval24) [inline lib/controllers/discussions_controller.dart:86]: hey can you make these database id and collection id dynamic by storing them in some variable. is there any reason for using them directly?

File: lib/views/screens/create_room_screen.dart [hunk 2]
```diff
@@ -34,10 +34,9 @@ class CreateRoomScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     createRoomController.initializeTagsController();
-    return WillPopScope(
-      onWillPop: () async {
+    return PopScope(
```
Comment (vrundraval24) [inline lib/views/screens/create_room_screen.dart:37]: BUG HERE.
Pressing Back button on Create New Room Screen (Audio Room) Closes the whole app.

**Review by vrundraval24 — CHANGES_REQUESTED** [review]
I cannot create New Audio Room, is this desired behavior?? the scheduled room is creating even without Date. Idk the implementation is still left or not. Or is this behavior desired. but please take a look.

and the BUG on pressing back button on Create Audio Room is also crashes app and on restart black screen appears. That need to be taken care of in priority.

**Comment by chandansgowda** [comment]
Update on this @Aarush-Acharya ?

**Comment by Aarush-Acharya** [comment]
@chandansgowda will have it done by today

**Comment by Aarush-Acharya** [comment]
Hey @chandansgowda  @vrundraval24  i have made the requested changes

**Review by vrundraval24 — APPROVED** [review]

**Comment by vrundraval24** [comment]
@chandansgowda 
Everything looks fine. 
Great job @Aarush-Acharya 🚀

**Comment by chandansgowda** [comment]
Great job @Aarush-Acharya 🚀

**Comment by Aarush-Acharya** [comment]
Thank you @chandansgowda, will have the other work done asap

**Pull request merged**